XenAdmin/Wizards/PatchingWizard/PatchingWizard_PrecheckPage.cs

```diff
@@ -501,6 +501,15 @@ protected virtual List<CheckGroup> GenerateCommonChecks(List<Host> applicableSer
                 }
                 if (pvChecks.Count > 0)
                     groups.Add(new CheckGroup(Messages.CHECKING_PV_GUESTS, pvChecks));
+
+                //protocol check - for each pool
+                var vSwitchChecks = (from Pool pool in SelectedPools.Where(p => Helpers.NaplesOrGreater(p.Connection))

```

Comment:
The check is not needed if the current version is Stockholm or greater (as it will always return "OK") so we can exclude Stockholm and greater hosts here.